password-verification.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
```

Comment:
Kindly work on this so it will look like the design

![image](https://user-images.githubusercontent.com/60205816/183466754-b2bab3b3-45c3-4c3d-aaf1-d11a803673be.png)